libs/designsystem/src/lib/components/index.ts

```diff
@@ -8,7 +8,8 @@ export { ButtonComponent } from './button/button.component';
 export * from './card';
 export * from './chart';
 export * from './stock-chart';
-export { CalendarComponent } from './calendar/calendar.component';
+export * from './calendar/calendar.component';
```

Comment:
Please align with existing code and add barrel file with public members that should be exposed in package:
```suggestion
export * from './calendar';
```